release-team/role-handbooks/docs/README.md

```diff
@@ -393,12 +406,11 @@ Create the PRs needed to roll the docs to the new version on the dev branch
 
     24 hours before the release, freeze the repo. No PRs allowed to merge AT ALL until the release PR has successfully merged.
 
-    ⚠️ **NEW:** This hasn't been tested, but after 1.14, we realized that not everyone would see our communications. We can temporarily modify k/website to be a "staging repo" which allows PROW to block all automatic functions (like merging).
+    ⚠️ **NEW:** After 1.16 release, we realized you only need to create an issue with ```blocker_label``` to prevent merging on a repo. Please check with **test-infra**  as this is not tested. You can find more information [here](https://github.com/kubernetes/test-infra/blob/master/prow/cmd/tide/config.md#merge-blocker-issues)
```

Comment:
What does ```blocker_label``` mean here?

Reply:
It means the label. I will update to make it clearer.